### ISSUE TITLE
Long loader phase during boot

### ISSUE THREAD

**Issue by FirelightFlagboy** [nix-community/lanzaboote#341]
I'm experiencing an issue with `lanzaboote`:

tldr: When I enable `lanzaboote` it drastically increase the boot time by `~1m30s`.

Upon booting with `lanzaboote`, I get the `systemd-boot` entries menu but just after an entry is selected, I get a black screen that stay around for `1m30s` and after that the boot sequence continue (messages start to be displayed to the screen).

I would like some help to understand why `lanzaboote` is slow during the loader phase (compared to `systemd-boot`)

## `systemd-analyze time`

With `systemd` as loader (`boot.loader.systemd-boot.enable == true && boot.lanzaboote.enable == false`):

```
Startup finished in 10.854s (firmware) + 2.409s (loader) + 383ms (kernel) + 5.706s (initrd) + 1.848s (userspace) = 21.202s 
multi-user.target reached after 1.847s in userspace.
```

With `lanzaboote` as loader (`boot.loader.systemd-boot.enable == false && boot.lanzaboote.enable == true`):

```
Startup finished in 4.722s (firmware) + 1min 47.311s (loader) + 387ms (kernel) + 23.883s (initrd) + 1.814s (userspace) = 2min 18.119s 
multi-user.target reached after 1.813s in userspace
```

Note that the duration of the loader phase a increased under `lanzaboote` to `+1m30s`

## Information

Lanzaboote: `v0.3.0` (rev 64b903ca87d18cef2752c19c098af275c6e51d63)

**Comment by FirelightFlagboy** [comment]
After testing with the latest commit on master (rev 850f27322239f8cfa56b122cc9a278ab99a49015) the issue seems to have disappear

```
Startup finished in 33.824s (firmware) + 2.913s (loader) + 394ms (kernel) + 4.710s (initrd) + 2.586s (userspace) = 44.428s 
multi-user.target reached after 2.585s in userspace.
```
